datajoint/declare.py

```diff
@@ -12,7 +12,7 @@
 
 UUID_DATA_TYPE = 'binary(16)'
 MAX_TABLE_NAME_LENGTH = 64
-CONSTANT_LITERALS = {'CURRENT_TIMESTAMP'}  # SQL literals to be used without quotes (case insensitive)
+CONSTANT_LITERALS = {'CURRENT_TIMESTAMP', 'CURRENT_TIMESTAMP(3)'}  # SQL literals to be used without quotes (case insensitive)
```

Comment:
hm... we should find a way to support `CURRENT_TIMESTAMP` with arbitrary precision rather than hardcode `(3)`.

Reply:
does MySQL itself do this? or is this future proofing?

Reply:
Yes but our parser does not.

Reply:
Do you mean as in a `my.cnf` setting? The default is second. If I declare a table and insert as follows:

``` MySQL
create table sample (dt timestamp(3));
insert into sample values (CURRENT_TIMESTAMP);
insert into sample values (NOW());
```

The result are only timestamps with second resolution.
``` MySQL
select dt from sample;
```
```
+-------------------------+
| dt                      |
+-------------------------+
| 2019-12-18 20:54:54.000 |
| 2019-12-18 20:54:56.000 |
+-------------------------+
2 rows in set (0.00 sec)
```

Reply:
I think we should modify the parsing code to allow specifying function calls as default values. Currently our parser will reject `CURRENT_TIMESTAMP(2)` for example.

Reply:
@dimitri-yatsenko Ah, I see your point. I have now updated the parsing to accommodate this.